patten_a05a.js

```diff
@@ -0,0 +1,33 @@
+/*Bryan Patten
+patten_a05a.js
+INFO 2124
+Lisa Thoendel
+01/19/2020*/
+
+let toDo = ["Wash dishes", "Clean garage", "Buy groceries", "Finish classwork"];
+/*Set four tasks in an array literal as part of a To Do list*/
+console.log(toDo);
+/*Write out each item of the array toDo in order*/
+toDo.unshift("Walk dog");
+/*Add new task to the beginning of the array toDo*/
+console.log(toDo);
+/*Wite out array toDo in order with new task at the beginning*/
+toDo.reverse();
+toDo.sort();
+console.log(toDo);
+/*Reverse array, sort it, then output the results*/
+toDo.includes("x");
```

Comment:
It would be great to see you do something with this - i.e. output that the array does or does not contain X.